compiler-plugin-tests/src/test/resources/ballerina_sources/generator_tests/25_local_graphql_service_object/service.bal

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2024 WSO2 LLC. (http://www.wso2.com) All Rights Reserved.
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/lang.runtime;
+import ballerina/graphql;
+
+public function main() returns error? {
+    graphql:Service gqlService = service object {
+        resource function get greeting() returns string {
+            return "Wubba lubba dub dub!!";
+        }
+    };
```

Comment:
Shall we add a test with complex scenario?

Reply:
Maybe something like this?
```ballerina
type Foo record {|
    string name;
    int age;
|};

public function main() returns error? {
    graphql:Service gqlService = service object {
        resource function get number() returns int {
            return 10;
        }
        
        resource function get greeting() returns string {
            return "Hello";
        }
        
        resource function get foo() returns Foo {
            return {
                name: "Bar",
                age: 100
            };
        }
    };
    graphql:Listener l = check new(9090);
    check l.attach(gqlService, "graphql");
    check l.'start();
    runtime:registerListener(l);
}
```

Reply:
+1. We need a test where we have a service declared with global variable and a local variable. We also need a test with a service declared using listener binding, and a locally defined service. We need to test whether the schema is generated correctly for all the services in the above scenarios. The point is, when we change the syntax tree, we need to make sure we do not remove the changes from the other services.